docs/setup-sensor/docker-multiple-connections.md

```diff
@@ -0,0 +1,147 @@
+
+## Prerequisites
+
+Before you begin, make sure you have:
+
+- A host machine with Docker installed and running. See the official [Docker installation guides](https://docs.docker.com/engine/install/). We're using the `macvlan` module. This module is currently only available on linux. But depending on your setup, it is possible to use the default network module, and skip the `macvlan` module in the config below.
+- All interfaces should be available for use on the host.
+- Subnet and gateway information for each network you want to monitor.
+- Docker Compose installed. See the [Docker Compose installation guide](https://docs.docker.com/compose/install/).
+- Good familiarity with using the command line or terminal on your host system.
+- Your host machine must be physically connected to the networks you wish to monitor.
+
+## Step 1: Prepare the Docker Compose file
+
+1.  Create a file named `docker-compose.yml` in a working directory.
+2.  Paste the following content exactly into the `docker-compose.yml` file:
+
+    ```yaml
+    services:
+    
+      orb_primary:
+        image: orbforge/orb:latest
+        container_name: orb-primary-isp
+        hostname: orb-primary-isp
+        restart: always
+        labels:
+          - "com.centurylinklabs.watchtower.enable=true"
+          - "com.centurylinklabs.watchtower.scope=orb"
+        networks:
+          isp_primary:
+            # optional, if you need to set a static ip, this is the place to do it
+            # if not, it will just use dhcp by default
+            # ipv4_address: 10.0.10.50
+            # ipv6_address: 
+        # optional: set a mac address
+        # mac_address: '02:42:0A:00:01:AA'
+        cap_add:
+          - NET_RAW
+        volumes:
+          - primary_isp_data:/root/.config/orb
+
+      # you can add as many orb sensors as you want to test for
+      orb_secondary:
+        image: orbforge/orb:latest
+        container_name: orb-secondary-isp
+        hostname: orb-secondary-isp
+        restart: always
+        labels:
+          - "com.centurylinklabs.watchtower.enable=true"
+          - "com.centurylinklabs.watchtower.scope=orb"
+        networks:
+          isp_secondary:
+            ipv4_address: 10.0.20.50
+        # mac_address: '02:42:0A:00:01:BB'
+        cap_add:
+          - NET_RAW
+        volumes:
+          - secondary_isp_data:/root/.config/orb
+
+      watchtower:
+        image: containrrr/watchtower
+        container_name: watchtower
+        restart: unless-stopped
+        volumes:
+          - /var/run/docker.sock:/var/run/docker.sock
+        command: --label-enable --scope orb --interval 86400
+
+    networks:
+      isp_primary:
+        driver_opts:
+          parent: enp8s0f1  # the interface you want to monitor
+  
+      isp_secondary:
+        driver: macvlan  # if the host interfaces have their own gateways already setup, you can skip this driver
```

Comment:
For my own understanding - is this only required if the 2 interfaces are on the same network and you need them to have different MACs?
Or is it required to make the parent/interface thing work at all?